cli/src/pcluster/models/cluster.py

```diff
@@ -309,8 +309,8 @@ class QueueNetworking(_BaseNetworking):
 
     def __init__(self, subnet_ids: List[str], placement_group: PlacementGroup = None, **kwargs):
         super().__init__(**kwargs)
-        self.subnet_ids = subnet_ids
-        self.placement_group = placement_group
+        self.subnet_ids = Param(subnet_ids)
```

Comment:
subnet_ids is the same case as security_groups and additional_security_groups. Type of placement_group is PlacementGroup. As I know, we only wrap raw types to Param, for this case. If we want to set default place_group, we could do this
```
default_placement_group = PlacementGroup(id)
default_placement_group.implied = True
self.placement_group = default_placement_group
``` 
If it is not default value, use it directly.

Reply:
Agreed on reverting the change for `PlacementGroup`; however I would avoid using the `implied` property in that way, as "implied" is by definition a property of a `Param` instance. Let's talk about this point if you want.